packages/consumption/src/modules/attributes/AttributesController.ts

```diff
@@ -979,6 +979,10 @@ export class AttributesController extends ConsumptionBaseController {
             return ValidationResult.error(ConsumptionCoreErrors.attributes.cannotSucceedAttributesWithASuccessor(predecessor.succeededBy.toString()));
         }
 
+        if (_.isEqual(parsedSuccessorParams.content, predecessor.content)) {
+            return ValidationResult.error(ConsumptionCoreErrors.attributes.successionMustChangeContent());
+        }
+
```

Comment:
Why'd you decided to put this error here?

One could also argue to throw the check after error `cannotSucceedAttributesWithDeletionInfo`. (If possible, errors should be thrown in such a way that correcting the cause of the error cannot lead directly to the next error being thrown.)